src/moscot/problems/generic/_generic.py

```diff
@@ -260,6 +262,7 @@ def prepare(
     @d.dedent
     def solve(
         self,
+        alpha: float = 0.5,
```

Comment:
can we set it to `1` by default, please? I.e. having pure `GW` per default.

Reply:
Done.